libethereum/BlockQueue.cpp

```diff
@@ -52,9 +52,10 @@ BlockQueue::BlockQueue():
 {
 	// Allow some room for other activity
 	unsigned verifierThreads = std::max(thread::hardware_concurrency(), 3U) - 2U;
+	verifierThreads = 1;
```

Comment:
Now the expression `std::max(...) - 2U` is not used anymore.

Maybe you can replace the initialization.  Maybe you can remove the variable `verifierThreads` altogether.